### PR TITLE
Clarify AMS dependencies

### PR DESCRIPTION
**Pull request by bf4**

### CODE REVIEW

**Comment by beauby** [comment]
:+1:

File: Gemfile [hunk 1]
```diff
@@ -3,17 +3,26 @@ source 'https://rubygems.org'
...
+if version == 'master'
+  gem 'rack', github: 'rack/rack'
+  git 'https://github.com/rails/rails.git' do
+    gem 'railties'
+    gem 'activesupport'
+    gem 'activemodel'
+    gem 'actionpack'
+    gem 'actionview'
+  end
+  gem 'rails-controller-testing', github: 'rails/rails-controller-testing'
```
Comment (bf4) [inline Gemfile]: Yeah, this is a little complicated.. but cool bundler feature, eh?

**Comment by bf4** [comment]
Basically, I want to avoid installing any gems in test that aren't necessary to use AMS and might contribute to unexpected behavior.  e.g. Right now AMS doesn't require ActionMailer or ActiveRecord, so why include it?

**Comment by joaomdmoura** [comment]
Nice @bf4, tks, merged

**Comment by bf4** [comment]
:smile:

**Pull request merged**